components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImpl.java

```diff
@@ -39,10 +41,38 @@ public HTTPGetFunctionImpl() {
     }
 
     @Override
-    public void httpGet(String epUrl, Map<String, Object> eventHandlers) {
+    public void httpGet(String epUrl, Object... params) {
+
+        Map<String, Object> eventHandlers;
+        Map<String, String> headers = new HashMap<>();
+
+        switch (params.length) {
```

Comment:
Check if params are not passed what happen

Reply:
When the params are not passed, default case occurs, which throws an `IllegalArgumentException`.